etcaterva/echaloasuerte
# Remove favorites functionality

Remove all references to favorites as we are planning on removing it in the future release of the new version of EAS.

Tested:
- Create a with a draw as favorite.
- Move to this branch.
- Log in and check favorites don't appear anymore.
